site/src/Librecores/ProjectRepoBundle/Controller/OrganizationController.php

```diff
@@ -377,8 +412,10 @@ public function denyAction($organizationName, $userName)
 
         // Update the organization membership
         $member = $this->getDoctrine()
-            ->getRepository('LibrecoresProjectRepoBundle:OrganizationMember')
-            ->findOneBy(['organization' => $o, 'user' => $user]);
+                       ->getRepository(
+                           'LibrecoresProjectRepoBundle:OrganizationMember'
+                       )
+                       ->findOneBy(['organization' => $o, 'user' => $user]);
```

Comment:
The original code was easy to read, the reformatted on is hard to read. Just leave the original one?